{"timestamp":"2025-08-01T17:21:19.938789","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-08-01T17:21:19.939312","level":"info","event":"Filling up the DagBag from /home/user/airflow/dags/my_first_dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-08-01T17:21:20.744069Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-01T17:21:20.744215Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-01T17:21:20.744473Z","level":"info","event":"Current task name:create_composite_before","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-01T17:21:20.744561Z","level":"info","event":"Dag name:ndvi_change_detection_pipeline_branching","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-01T17:21:20.752179Z","level":"info","event":"Date string from XCom for before composite: 2025-01-04","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-01T17:21:20.757135","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"PermissionError","exc_value":"[Errno 13] Permission denied: '/output'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/user/airflow_venv_clean/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":877,"name":"run"},{"filename":"/home/user/airflow_venv_clean/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1164,"name":"_execute_task"},{"filename":"/home/user/airflow_venv_clean/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/user/airflow_venv_clean/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":217,"name":"execute"},{"filename":"/home/user/airflow_venv_clean/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":240,"name":"execute_callable"},{"filename":"/home/user/airflow_venv_clean/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/home/user/airflow/ndvi_utils.py","lineno":728,"name":"create_composite_before_wrapper"},{"filename":"<frozen os>","lineno":225,"name":"makedirs"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-08-01T17:21:20.783040Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-01T17:21:20.783207Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-01T17:21:20.783268Z","level":"info","event":"Task:<Task(PythonOperator): create_composite_before>","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-01T17:21:20.783296Z","level":"info","event":"Failure caused by [Errno 13] Permission denied: '/output'","chan":"stdout","logger":"task"}
